qemu/tests/cfg/smt_test.cfg

```diff
@@ -0,0 +1,45 @@
+- smt_test:
+    virt_test_type = qemu
+    type = smt_test
+    start_vm = no
+    kill_vm_on_error = yes
+    only x86_64, i386
+    only HostCpuVendor.amd
+    cpu_model_flags += ",enforce"
+    auto_cpu_model = no
+    smp = 2
+    vcpu_maxcpus = ${smp}
+    vcpu_threads = 2
+    vcpu_cores = 1
+    vcpu_sockets = 1
+    test_negative_thread = yes
+    check_flag_cmd = lscpu | grep Flags | awk -F ':'  '{print $2}'
```

Comment:
How about to define a default `check_flag_cmd` in cpu_utils`? I don't think it's necessary to let different scripts define it, in most cases it should be the same.

Reply:
Sure, updated.